.github/scripts/bash/plugins/pluginTests.sh

```diff
@@ -0,0 +1,84 @@
+        else
+            ./console tests:run-ui --store-in-ui-tests-repo --persist-fixture-data --assume-artifacts --core --extra-options="$UITEST_EXTRA_OPTIONS"
+        fi
+    elif [ "$TEST_SUITE" = "AllTests" ]
+    then
+        exit_code=0
+
+        if [ "$ALLTEST_EXTRA_OPTIONS" = "--run-first-half-only" ]
+        then
+            echo "Executing tests in test suite UnitTests"
+            ./vendor/phpunit/phpunit/phpunit --configuration ./tests/PHPUnit/phpunit.xml --testsuite UnitTests --colors $PHPUNIT_EXTRA_OPTIONS || exit_code=$?
+            echo "Executing tests in test suite SystemTests for Plugins"
+            ./vendor/phpunit/phpunit/phpunit --configuration ./tests/PHPUnit/phpunit.xml --testsuite SystemTestsPlugins --colors $PHPUNIT_EXTRA_OPTIONS || exit_code=$?
+            echo "Executing tests in test suite SystemTests for Core"
+            ./vendor/phpunit/phpunit/phpunit --configuration ./tests/PHPUnit/phpunit.xml --testsuite SystemTestsCore --colors $PHPUNIT_EXTRA_OPTIONS || exit_code=$?
+        elif [ "$ALLTEST_EXTRA_OPTIONS" = "--run-second-half-only" ]
+        then
+            echo "Executing tests in test suite IntegrationTests"
+            ./vendor/phpunit/phpunit/phpunit --configuration ./tests/PHPUnit/phpunit.xml --testsuite IntegrationTests --colors $PHPUNIT_EXTRA_OPTIONS || exit_code=$?
+        else
+            ./console tests:run --options="--colors" || exit_code=$?
+        fi
+
+        exit $exit_code
+    else
+        if [ -n "$PLUGIN_NAME" ]
+        then
+            ./vendor/phpunit/phpunit/phpunit --configuration ./tests/PHPUnit/phpunit.xml --colors --testsuite $TEST_SUITE --group $PLUGIN_NAME --coverage-clover $PIWIK_ROOT_DIR/build/logs/clover-$PLUGIN_NAME.xml $PHPUNIT_EXTRA_OPTIONS | tee phpunit.out
+        else
+            ./vendor/phpunit/phpunit/phpunit --configuration ./tests/PHPUnit/phpunit.xml --testsuite $TEST_SUITE --colors $PHPUNIT_EXTRA_OPTIONS | tee phpunit.out
+        fi
+
+        exit_code="${PIPESTATUS[0]}"
+        if [ "$exit_code" -ne "0" ]; then
+            exit $exit_code
+        elif grep "No tests executed" phpunit.out; then
+            exit 1
+        else
+            exit 0
+        fi
+    fi
+else
+    if [ "$COVERAGE" = "Unit" ]
+    then
+        echo "Executing tests in test suite UnitTests..."
+        ./vendor/phpunit/phpunit/phpunit --configuration ./tests/PHPUnit/phpunit.xml --testsuite UnitTests --colors --coverage-clover $TRAVIS_BUILD_DIR/build/logs/clover-unit.xml $PHPUNIT_EXTRA_OPTIONS || true
+    elif [ "$COVERAGE" = "Integration" ]
+    then
+        echo "Executing tests in test suite IntegrationTests..."
+        ./vendor/phpunit/phpunit/phpunit --configuration ./tests/PHPUnit/phpunit.xml --testsuite IntegrationTests --colors --coverage-clover $TRAVIS_BUILD_DIR/build/logs/clover-integration.xml $PHPUNIT_EXTRA_OPTIONS || true
+    fi;
```

Comment:
I would say we can remove all that code coverage stuff. It wasn't used in a long time and if we ever consider using that again, I think it's better to rethink that completely.